### ISSUE TITLE
[v0.10.2][k8s-v1.22.2] deploy manifest fail due to args `--log-level=info` issue 

### ISSUE THREAD

**Issue by sysnet4admin** [metallb/metallb#956]
Hi metallb team, 
I hope you doing well. 

Here is my configurations 
```
k8s_V = '1.22.2'           # Kubernetes 
docker_V = '20.10.8-3.el7' # Docker  
ctrd_V = '1.4.9-3.1.el7'   # Containerd 
metallb_V = 'v0.10.2'
CNI = Calico v3.17.1
kube-proxy = iptables 
```
I have encountered to deploy latest metallb in my VM groups like below: 
https://github.com/metallb/metallb/blob/main/manifests/metallb.yaml

```bash 
[root@m-k8s ~]# k get po -n metallb-system
NAME                          READY   STATUS             RESTARTS         AGE
controller-5897c45fb7-jj26v   0/1     CrashLoopBackOff   8 (3m13s ago)    27m
speaker-2n7rz                 0/1     CrashLoopBackOff   8 (4m47s ago)    21m
speaker-9fs7k                 0/1     CrashLoopBackOff   7 (4m59s ago)    16m
speaker-k2r88                 0/1     CrashLoopBackOff   5 (2m40s ago)    6m20s
speaker-vlvp4                 0/1     Error              10 (5m15s ago)   27m
```
and `logs` 
```bash 
[root@m-k8s ~]# k logs -n metallb-system speaker-2n7rz
flag provided but not defined: -log-level
Usage of /speaker:
  -config string
        Kubernetes ConfigMap containing MetalLB's configuration (default "config")
  -host string
        HTTP host address (default "192.168.1.101")
  -kubeconfig string
        absolute path to the kubeconfig file (only needed when running outside of k8s)
  -ml-bindaddr string
        Bind addr for MemberList (fast dead node detection) (default "192.168.1.101")
  -ml-bindport string
        Bind port for MemberList (fast dead node detection)
  -ml-labels string
        Labels to match the speakers (for MemberList / fast dead node detection) (default "app=metallb,component=speaker")
  -ml-secret-key string
        Secret key for MemberList (fast dead node detection) (default "nP6Mru9a145ru4Kgzz56pKFNEjPP7OXYrowc4w2nu2eVUH7m/9ol+OHpPu+0POSb\n/x1DGGLSTGiFJ4izjqM1+IuFfv8oDFYL16JpmjyAATvrq3+m2EwJP7BQHFiBAJpd\ncEgAKWsaBbiARAUCLQJZaRHyhzX5Y72Hge4cuap6Wjc=")
  -namespace string
        config file and speakers namespace
  -node-name string
        name of this Kubernetes node (spec.nodeName) (default "w1-k8s")
  -port int
        HTTP listening port (default 7472)

```

Thus I removed `--log-level=info` args columns. 
https://github.com/metallb/metallb/blob/main/manifests/metallb.yaml#L341
https://github.com/metallb/metallb/blob/main/manifests/metallb.yaml#L422

And deploy successfully. 
```bash 
[root@m-k8s ~]# k apply -f https://raw.githubusercontent.com/sysnet4admin/IaC/master/manifests/svc/metallb-0.10.2.yaml
namespace/metallb-system created
Warning: policy/v1beta1 PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25+
podsecuritypolicy.policy/controller created
podsecuritypolicy.policy/speaker created
serviceaccount/controller created
serviceaccount/speaker created
clusterrole.rbac.authorization.k8s.io/metallb-system:controller created
clusterrole.rbac.authorization.k8s.io/metallb-system:speaker created
role.rbac.authorization.k8s.io/config-watcher created
role.rbac.authorization.k8s.io/pod-lister created
role.rbac.authorization.k8s.io/controller created
clusterrolebinding.rbac.authorization.k8s.io/metallb-system:controller created
clusterrolebinding.rbac.authorization.k8s.io/metallb-system:speaker created
rolebinding.rbac.authorization.k8s.io/config-watcher created
rolebinding.rbac.authorization.k8s.io/pod-lister created
rolebinding.rbac.authorization.k8s.io/controller created
daemonset.apps/speaker created
deployment.apps/controller created
[root@m-k8s ~]# ^C
[root@m-k8s ~]# k get po -n metallb-system
NAME                          READY   STATUS    RESTARTS   AGE
controller-6878df9869-x4d9q   1/1     Running   0          5m6s
speaker-27h5j                 1/1     Running   0          5m6s
speaker-fj7r9                 1/1     Running   0          5m6s
speaker-mwsrz                 1/1     Running   0          5m6s
speaker-w42r2                 1/1     Running   0          5m6s
```

I am not sure it is my some dependency whether or not. 
Hopefully someone help to this diagnostic. 
And if it is some dependency, I wish it will update as remark or docs. 

Hoon Jo.

**Comment by champtar** [comment]
Use the manifest from the 0.10 branch instead of main

**Comment by sysnet4admin** [comment]
@champtar Thank you for your comment. 
I've checked 0.10 ver. there is no `--log-level=info`. 
It is some dependency or alpha status?

I really appropriate your any kind of !! response. :)

**Comment by champtar** [comment]
main is the branch where development happens, and this new flag was added after 0.10.2 was released, so you are trying to use a new manifest with an unreleased feature but using older image. The website normally points to the latest released manifest (so 0.10.2).

**Comment by sysnet4admin** [comment]
Thank you for your kindness message. 
Have a nice and safety day. :)
